test/e2e/cypress/e2e/databases_overview.cy.js

```diff
@@ -18,4 +18,19 @@ context('Databases Overview', () => {
       cy.contains(hdqDatabase.sid).should('not.exist');
     });
   });
+
+  describe('Restoration', () => {
+    const databaseToRestore = {
```

Comment:
We have seen some recent failure on Databases and SAP systems restoration. The instances belonging to them were not recovered properly after a browser refresh.
Maybe we should consider those events.

Reply:
I've added the test and indeed it fails. Oddly enough I can't consistently reproduce the bug manually, but cypress catches it.